### ISSUE TITLE
oelint.vars.pathhardcode in comments

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#74]
```
# /usr/lib is a place my libs will call home
```

is a valid snippet and shouldn't be complained about
